x-pack/plugin/wildcard/src/main/java/org/elasticsearch/xpack/wildcard/mapper/WildcardFieldMapper.java

```diff
@@ -844,7 +845,36 @@ public Query existsQuery(QueryShardContext context) {
 
         @Override
         public Query termQuery(Object value, QueryShardContext context) {
-            return wildcardQuery(BytesRefs.toString(value), MultiTermQuery.CONSTANT_SCORE_REWRITE, context);
```

Comment:
Isn't it the same problem with `prefix` query ? Maybe we can just escape `?` and `*` in the value and continue to use the `wildcardQuery` ?

Reply:
Good point. I'll try that approach.